### PR TITLE
Wacom.WacomTabletDriver version 6.3.45-1

### PR DESCRIPTION
**Pull request by JasonCook599**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40554)

### CODE REVIEW

**Comment by JasonCook599** [comment]
The exit code appears to have no bearing on the success of the install. On my system, it has exited with codes 1 and 2 on a successful install. It has exited successfully with no exit code after an aborted install.

File: manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Wacom.WacomTabletDriver
+PackageVersion: 6.3.45-1
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
```
Comment (ItzLevvie) [inline manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml:8]:
```suggestion
  SilentWithProgress: /s
InstallerSuccessCodes:
- 2
```

Comment (ItzLevvie) [reply manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml:8]: image

Looks like `2` is a success:
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
I was going to suggest the same as Levvie, however i don't think it installed correctly with the /Q switch, No Start Menu shortcuts and the executable files in `C:\Program Files\Tablet` didn't work, when running it manually it creates Start Menu shortcuts AND `C:\Program Files\Tablet` looks completely different

File: manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: Wacom.WacomTabletDriver
+PackageVersion: 6.3.45-1
+PackageLocale: en-US
+Publisher: Wacom Technology, Corp.
+PublisherUrl: https://www.wacom.com/
+PublisherSupportUrl: https://www.wacom.com/en-ca/support
+PrivacyUrl: https://www.wacom.com/privacy
+Author: Wacom Technology, Corp.
+PackageName: Wacom Tablet Driver
```
Comment (ItzLevvie) [inline manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.locale.en-US.yaml:12]:
```suggestion
PackageName: Wacom Tablet
```
to match Control Panel.

File: manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Wacom.WacomTabletDriver
+PackageVersion: 6.3.45-1
+PackageLocale: en-US
+Publisher: Wacom Technology, Corp.
```
Comment (ItzLevvie) [inline manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.locale.en-US.yaml:7]:
```suggestion
Publisher: Wacom Technology Corp.
```
to match Control Panel.

File: manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+PublisherUrl: https://www.wacom.com/
+PublisherSupportUrl: https://www.wacom.com/en-ca/support
+PrivacyUrl: https://www.wacom.com/privacy
+Author: Wacom Technology, Corp.
+PackageName: Wacom Tablet Driver
+PackageUrl: https://www.wacom.com/en-ca/sandbox/drivers#latest
+License: Wacom Technology, Corp. 2019
+LicenseUrl: https://www.wacom.com/terms-of-use
+Copyright: Wacom Technology, Corp.
+ShortDescription: Wacom Tablet Driver
```
Comment (ItzLevvie) [inline manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.locale.en-US.yaml:17]:
```suggestion
ShortDescription: Drivers for Wacom Tablet.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml (hunk 1 above)
Comment (JasonCook599) [reply manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml:8]: Thanks for this. I wasn't clear on how to resolve the issue with the exit code.

**Comment by JasonCook599** [comment]
> I was going to suggest the same as Levvie, however i don't think it installed correctly with the /Q switch, No Start Menu shortcuts and the executable files in `C:\Program Files\Tablet` didn't work, when running it manually it creates Start Menu shortcuts AND `C:\Program Files\Tablet` looks completely different

Where are you getting the /Q switch from? The /s switch is used for a silent install. On my system, this does create the correct files in `C:\Program Files\Tablet` and the start menu shortcuts.

File: manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+PackageIdentifier: Wacom.WacomTabletDriver
+PackageVersion: 6.3.45-1
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
+InstallerSuccessCodes:
+- 1
+- 2
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml:13]:
```suggestion
- Architecture: x64
```
It's failing on x86 VM; so it's better to use `x64` here:
image
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/40554/files#r783174898

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**